test/functional/test_framework/blocktools.py

```diff
@@ -44,14 +44,15 @@
 # From BIP141
 WITNESS_COMMITMENT_HEADER = b"\xaa\x21\xa9\xed"
 
-def create_block(hashprev, coinbase, ntime=None):
+def create_block(hashprev, coinbase, block_height, ntime=None):
```

Comment:
Now that `block_height` is provided here, suggest making `coinbase` optional since almost all uses just call `create_coinbase(block_height)`. Probably doesn't matter much unless/until this goes upstream...

Reply:
I'd have to implement a CScriptNum encoder in python.... sigh I'll do it :)

Reply:
Huh? `create_coinbase` (and `serialize_script_num`) already exists..

Reply:
de-serialize*. I want to grab it directly from the coinbase.